CDP4Dal/ISession.cs

```diff
@@ -186,18 +186,47 @@ public interface ISession
         Task Open();

```

Comment:
my preference would be to add a parameter here that is not mandatory and true by default. Something like:

`Task Open(bool activeMessageBus = true);`

Then we do not need an extra method that in my opinion will cause confusion.

@alexatrhea : what do you think

Reply:
yes agreed